ca_gazebo/launch/create_cooperative_two_rooms.launch

```diff
@@ -0,0 +1,13 @@
+<launch>
```

Comment:
This file is not necessary. Add the dock to the empty world and spawn it with an environment variable `DOCK`. You will also need to declare the initial pose for each environment.